CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 1.6.0 (TBD)
```

Comment:
Question: why not include this in `1.5.0`?

Reply:
Just too busy to get it out on time. I can target `1.5.1`